.gitignore

```diff
@@ -31,4 +31,5 @@ tests/tmp.*
 *.charm
 *.bundle
 *.log
-dqlite-deps.tar.bz2
\ No newline at end of file
+dqlite-deps.tar.bz2
+docs/
```

Comment:
We should drop this. This should only be in 3.5

Reply:
Done

Reply:
This will need to be fixed in 3.6 as well oops